<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QWidget" name="container" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QStackedWidget" name="panelW">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="playW">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(164,  36, 100)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
             </property>
             <item row="3" column="2">
              <widget class="QLabel" name="boardSize">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: rgb(255, 0, 0);
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QLabel" name="numberPlayersL">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: #FFFFFF;
}
</string>
               </property>
               <property name="text">
                <string>Cantidad De Jugadores:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="forL">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: #FFFFFF;
}
</string>
               </property>
               <property name="text">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="columsL">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: #FFFFFF;
}
</string>
               </property>
               <property name="text">
                <string>Columnas</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLineEdit" name="columnSize">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    background-color: #1E1E1E;
    color: #00FFAA;
    border: 2px solid #00FFAA;
    border-radius: 6px;
    padding: 6px;
    font-size: 16px;
}
</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="rowsL">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: #FFFFFF;
}
</string>
               </property>
               <property name="text">
                <string>Filas</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="3">
              <widget class="QTableWidget" name="tableWidget">
               <property name="styleSheet">
                <string notr="true">QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    padding: 4px;
    border: 1px solid #34495e;
    font-weight: bold;
}

QTableWidget {
    background-color: #ecf0f1;
    alternate-background-color: #dfe6e9;
    selection-background-color: #3498db;
    selection-color: white;
color: black;
    border: 1px solid #95a5a6;
    gridline-color: #bdc3c7;
}

QTableWidget QTableCornerButton::section {
    background-color: #2c3e50;
    border: 1px solid #34495e;
}
</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLineEdit" name="rowSize">
               <property name="styleSheet">
                <string notr="true">QLineEdit {
    background-color: #1E1E1E;
    color: #00FFAA;
    border: 2px solid #00FFAA;
    border-radius: 6px;
    padding: 6px;
    font-size: 16px;
}
</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLabel" name="sizeL">
               <property name="styleSheet">
                <string notr="true">QLabel {
    font-size: 26px;
    font-weight: bold;
    color: #FFFFFF;
}
</string>
               </property>
               <property name="text">
                <string>Tama√±o de Tablero:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="playersB">
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: #007ACC;
    color: white;
    font-size: 18px;
    font-weight: bold;
    border-radius: 10px;
    padding: 12px 24px;
    border: none;
}

QPushButton:hover {
    background-color: #005F9E;
}

QPushButton:pressed {
    background-color: #003F6B;
}
</string>
               </property>
               <property name="text">
                <string>Ingresar Jugadores</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="play">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    background-color: #007ACC;
    color: white;
    font-size: 18px;
    font-weight: bold;
    border-radius: 10px;
    padding: 12px 24px;
    border: none;
}

QPushButton:hover {
    background-color: #005F9E;
}

QPushButton:pressed {
    background-color: #003F6B;
}
</string>
             </property>
             <property name="text">
              <string>Jugar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="main">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 170, 255)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="optionsW" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 127)
</string>
         </property>
         <widget class="QPushButton" name="playB">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>90</y>
            <width>201</width>
            <height>81</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #4CAF50;
    color: white;
    font-size: 24px;
    font-weight: bold;
    border-radius: 12px;
    padding: 20px 40px;
    border: 2px solid #388E3C;
}

QPushButton:hover {
    background-color: #45a049;
}

QPushButton:pressed {
    background-color: #2E7D32;
    border-style: inset;
}
</string>
          </property>
          <property name="text">
           <string>Jugar</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="nav" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>playB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
